common/game/mode/fugitive/networking/FugitivePlayerDataUtility.gd

```diff
@@ -0,0 +1,31 @@
+extends Node
+
+
+func get_stats(playerData: PlayerData) -> Dictionary:
+	if !playerData.player_data_dictionary.has("stats"):
+		print("STATS - No stats for player %s" % playerData.get_id())
+		playerData.player_data_dictionary.stats = {}
+	
+	return playerData.player_data_dictionary.stats
+	
+func increment_stat_for_player_id(playerId: int, identifier: String):
+	print("STATS - Incrementing %s for player %s" % [identifier, playerId])
+	increment_stat(GameData.get_player(playerId), identifier)
+	
+func increment_stat(playerData: PlayerData, identifier: String):
```

Comment:
Maybe this accepts an argument for how much to increment by, and we default it to 1 `, ammount: int = 1`

No need to hold up this PR for it, but might be a nice feature going forward.

Reply:
Oh yeah, I dig that - let's add that if there's ever a stat that can go up by more than one at a time.  (right now I think there's only things that happen in response to player actions, or that can only happen once a game)